### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by b-buchkevych**
- [DEMO LINK](https://b-buchkevych.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://b-buchkevych.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+  line-height: 40px;
+  text-align: center;
+  text-transform: uppercase;
+}
+
+.card__buy:hover {
+  color: var(--main-color);
+  background: #fff;
+  border: 1px solid var(--main-color);
+  border-radius: 5px;
```
Comment (AlisaVlasova) [inline src/style.css:125]: radius does not change

**Review by AlisaVlasova — CHANGES_REQUESTED** [review]

**Review by etojeDenys — CHANGES_REQUESTED** [review]
create only files for element `blocks` (e.g: card, stars, page)

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+  font-size: 14px;
+  line-height: 40px;
+  text-align: center;
+  text-transform: uppercase;
+}
+
+.card__buy:hover {
+  color: var(--main-color);
+  background: #fff;
+  border: 1px solid var(--main-color);
```
[the remark this answers is not in the document]
Comment (tetiana-ivanchenko) [reply src/style.css:124]: fix it https://github.com/mate-academy/layout_product-cards/pull/2786/files#r1143678556

File: src/style.css (hunk 1 above)
Comment (tetiana-ivanchenko) [reply src/style.css:125]: fix it https://github.com/mate-academy/layout_product-cards/pull/2786/files#r1143679301

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+
+.card__price-value {
+  font-size: 16px;
+  line-height: 18px;
+  font-weight: 700;
+  color: var(--text-main-color);
+}
+
+.card__buy {
+  height: 40px;
```
[the remark this answers is not in the document]
Comment (tetiana-ivanchenko) [reply src/style.css:108]: fi it https://github.com/mate-academy/layout_product-cards/pull/2786/files#r1143677799

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+  --text-main-color: #060b35;
+  --text-second-color: #616070;
+}
+
+body,
+p {
+  margin: 0;
+}
+
+.card {
```
[the remark this answers is not in the document]
Comment (tetiana-ivanchenko) [reply src/style.css:17]: fit it https://github.com/mate-academy/layout_product-cards/pull/2786/files#r1143676496

**Review by tetiana-ivanchenko — CHANGES_REQUESTED** [review]
re-request review only if you fixed all comments? if you have sny questions - feel free to ask in the corresponding fe_chat

**Review by tetiana-ivanchenko — CHANGES_REQUESTED** [review]

**Pull request closed without merging**